_pages/en/web-services-access.md

```diff
@@ -1,63 +1,57 @@
 ---
 layout: default
-title: Granting access to Web tools at ESDC
+title: Granting More Open Access to the Internet for IITB Employees
 ref: web-services-access
 lang: en
 status: posted
 sections: Work In Progress
 permalink: /web-services-access.html
 ---
 
-## Granting access to Web tools at ESDC
+## Granting More Open Access to the Internet for IITB Employees
 
-### Purpose
+### Summary
 
-To seek endorsement for granting access to more Web tools or Software as a Service (SaaS) for IITB employees, as a step toward broader open access to the Internet for all ESDC employees.
+Departments are leveraging Web tools or Software as a Service (SaaS) to collaborate and engage with citizens and public administrations both in Canada and across the globe.
+For example, the latest Open Government Plan and Service and Digital Target Architecture from TBS were shared as drafts for consultation and comments on Google Docs.
+Many teams are also using Slack or GCcollab Message to communicate internally and across departments.
+The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
 
-### Background
+In order to improve ESDC's ability to leverage current Web tools, a more open access to Web tools or Software as a Service (SaaS) for IITB employees, as a step toward broader open access to the Internet for more ESDC employees.
 
-The upcoming [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) replaces the [Policy on Acceptable Network and Device Use (PANDU)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27122) but keeps the intent to "provide open access to Internet tools and services to support public servants in their work duties, enhance collaboration and networking with their peers".
-The policy applies only to unclassified interactions (data, information, code, etc.).
+As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, the risk would be minimal.
```

Comment:
This won't stand the IT Security test. It's vague and easily dismantled by any IT Security stats to show cases where IITB staff behaved inadequately (bad coders, USB key inputted in laptops, sensitive emails sent, testing using real client data instead of anonymous ones). Instead recommend changing to:

- IITB employees produce the software and tools needed by ESDC Staff to perform their duties. IITB employees work description do not require them to process client files like ESDC front staff do. Producing the software and tools require important amount of collaboration between teams in order to remain responsive to client's need
- The Government of Canada is recognizing that Security must be handled with user experience in mind, and that Government employees require the right tools to do their work otherwise the Government is at risk at falling behind citizen expectations. This is highlighted in the following policy instruments:
-- [Direction on Enabling Web Access to Web Services](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32588)
-- [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601), in particular requirement #4.4.11.1 referencing the Acceptable and Unacceptable network device use
-- [Values and Ethics of the Public Service](https://www.canada.ca/en/government/publicservice/values.html)
-- [ESDC's Code of Conduct](https://www.canada.ca/en/employment-social-development/corporate/code-conduct.html), in particular [Section D) Stewardship on the use of Electronic Networks](https://www.canada.ca/en/employment-social-development/corporate/code-conduct.html#a3.4)